### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by donothingloop**
Fixed paths to python scripts as the bin directory was removed.
Also added instruction to set executable permission.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,23 +38,24 @@ Curtesy of [this](http://googleresearch.blogspot.ru/2015/11/computer-respond-to-
 
     git clone git@github.com:nicolas-ivanov/tf_seq2seq_chatbot.git
     cd tf_seq2seq_chatbot
-    bash bin/setup.sh
+    chmod +x setup.sh
```
Comment (nicolas-ivanov) [inline README.md:41]: do we really need this line?
seems to me `bash setup.sh` gonna do the job all the same

**Comment by donothingloop** [comment]
Yes, you're right. The chmod +x line is unnecessary.

I'll change the pull request.

**Comment by donothingloop** [comment]
I have opened another pull request.
https://github.com/nicolas-ivanov/tf_seq2seq_chatbot/pull/5

**Comment by nicolas-ivanov** [comment]
Great! Next time feel free to just add a commit with your amendments, I think it's a bit easier than closing a pull-request and opening a new one ;)

**Comment by donothingloop** [comment]
I like to keep the commit history clean :)
It's a bit strange when there is a commit with the title "Updated README.md" and another one with "Removed unnecessary command" and I was a too lazy to amend the commit.

**Comment by nicolas-ivanov** [comment]
Ok, I see your point )

**Pull request closed without merging**